index.src.html

```diff
@@ -349,6 +348,54 @@
           view</dfn> which is a double. It represents the vertical angle of
           view in degrees.
         </p>
+        <p>
+          The data type of a <a>depth map</a> is 16-bit unsigned integer. The
+          algorithm to <dfn>convert the depth map value to grayscale</dfn>,
+          given a depth map value <var>d</var>, is as follows:
+        </p>
+        <ol>
+          <li>Let <var>near</var> be the the <a>near value</a>.
+          </li>
+          <li>Let <var>far</var> be the the <a>far value</a>.
+          </li>
+          <li>Apply the <a>rules to convert using range inverse</a> to
```

Comment:
Only if the destination format does not match the internal format, 16-bit depth. Otherwise, conversion is required. If we had a 16-bit sink, there is no need to use a predefined conversion (`RangeLinear` or `RangeInverse`).